tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/manifest.ini

```diff
@@ -26,3 +26,5 @@ carrier = true
 [test_lockscreen_notification.py]
 
 [test_lockscreen_wake_with_notification.py]
+
+[test_lockscreen_time_check.py]
```

Comment:
Actually I can get the automatic setting via wifi, so the phone does not necessarily have the sim card